treescript/tests/data/test_private_key.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Nit: I wonder if it's worth using an ED25519 key in the tests. This way, we're sure to test with the best recommended cryptographic keys. I'd recommend using such a key in staging and production, if that's not already the case.

Reply:
Unfortunately we don't get a choice. Github generates the key for us and it uses RSA. I generated this test key from the Github app (and immediately deleted it from the app afterward so it's no longer valid).